lib/main.dart

```diff
@@ -0,0 +1,39 @@
+//import 'dart:html';
+
+import 'package:flutter/material.dart';
+import 'movie.dart';
+import 'movie_main_widget.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      top: true,
+      bottom: true,
+      left: true,
+      right: true,
```

Comment:
SafeArea default values are true so in this case you dont need to use top , bottom , left and right

Reply:
Done